src/content/api/wallets.mdx

```diff
@@ -0,0 +1,285 @@
+      [Asset Type](https://docs.centrapay.com/api/asset-types) id used by payment option asset types.
+    </Property>
+
+    <Property name="settlement" type="boolean" experimental>
+      `true` if the Wallet is a [Settlement Wallet](#settlement-wallets).
+    </Property>
+  </Properties>
+
+  <CodePanel slot="code-examples" title="Request" method="POST" path="/api/wallets">
+    ```bash
+    curl -X POST https://service.centrapay.com/api/wallets \
+      -H "X-Api-Key: $api_key" \
+      -H "Content-Type: application/json" \
+      -d '{
+        "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+        "ledgerId": "centrapay.nzd.main",
+        "settlement": "true"
+      }'
+    ```
+  </CodePanel>
+
+  <CodePanel slot="code-examples" title="Response">
+    ```json
+    {
+      "id": "WRhAxxWpTKb5U7pXyxQjjY",
+      "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+      "category": "money",
+      "type": "centrapay.nzd.main",
+      "liveness": "main",
+      "description": "NZD",
+      "createdAt": "2021-01-01T00:00:00.000Z",
+      "status": "active",
+      "currency": "NZD",
+      "balance": "0",
+      "availableBalance": "6000",
+      "settlement": true
+    }
+    ```
+  </CodePanel>
+
+  ### Errors
+  <Properties>
+    <Error code="403" message="ACCOUNT_MAX_WALLETS_REACHED">
+      The maximum number of wallets for the given ledger has been reached.
+    </Error>
+  </Properties>
+
+</Endpoint>
+
+---
+
+<Endpoint
+  method="GET"
+  path="/api/wallets"
+>
+  ## List Wallets
+
+  This endpoint allows you to list authorized Wallets.
+
+  ### Attributes
+  No attributes.
+
+  <CodePanel slot="code-examples" title="Request" method="GET" path="/api/wallets">
+    ```bash
+    curl https://service.centrapay.com/api/wallets \
+      -H "X-Api-Key: $api_key"
+    ```
+  </CodePanel>
+
+  <CodePanel slot="code-examples" title="Response">
+    ```json
+    [
+      {
+        "id": "WRhAxxWpTKb5U7pXyxQjjY",
+        "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+        "ledgerId": "centrapay.nzd.main",
+        "currency": "NZD",
+        "balance": "2000",
+        "availableBalance": "6000"
+      },
+      {
+        "id": "NQ1yeromwnWPD2hY41L2yS",
+        "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+        "ledgerId": "centrapay.nzd.test",
+        "currency": "NZD",
+        "balance": "20",
+        "availableBalance": "6000"
+      }
+    ]
+    ```
+  </CodePanel>
+
+</Endpoint>
+
+---
+
+<Endpoint
+  method="GET"
+  path="/api/wallets/{walletId}/transactions"
+>
+  ## List Wallet Transactions
+
+  This endpoint allows you to list Wallet Transactions.
+
+  ### Attributes
+  No attributes.
+
+  <CodePanel slot="code-examples" title="Request" method="GET" path="/api/wallets/{walletId}/transactions">
+    ```bash
+    curl https://service.centrapay.com/api/wallets/WRhAxxWpTKb5U7pXyxQjjY/transactions \
+      -H "X-Api-Key: $api_key"
+    ```
+  </CodePanel>
+
+  <CodePanel slot="code-examples" title="Response">
```

Comment:
There are a lot of examples in the existing documentation. I was going to do multiple items but JSON doesn't support comments so could be confusing.

Reply:
Can we include all of the examples as separate code panels with the example name as the title?

Reply:
@AndyClifford we could but I had discussed this with Tom and the jesters and the patterns was going to be one response code panel per endpoint, with the idea that our guides should be better.